junit-platform-launcher/src/main/java/org/junit/platform/launcher/core/LauncherFactory.java

```diff
@@ -39,7 +46,11 @@
 	 * @throws PreconditionViolationException if no test engines are detected
```

Comment:
The Javadoc must also mention that TELs which are dynamically detected will be automatically registered in the Launcher.